recipe/meta.yaml

```diff
@@ -26,12 +26,13 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python
+    - python >=3.6
 
 test:
   imports:
     - bitarray
   requires:
+    - python <3.10
     - pip
   commands:
     - pip check
```

Comment:
The license should be `license: PSF-2.0`, see https://spdx.org/licenses/PSF-2.0.html